--- bfd/Makefile.in.orig	2021-01-21 21:16:29.473000000 +0000
+++ bfd/Makefile.in	2021-01-21 21:17:15.470000000 +0000
@@ -461,7 +461,7 @@
 ACLOCAL_AMFLAGS = -I . -I .. -I ../config
 INCDIR = $(srcdir)/../include
 CSEARCH = -I. -I$(srcdir) -I$(INCDIR)
-SUBDIRS = doc po
+SUBDIRS = po
 bfddocdir = doc
 libbfd_la_LDFLAGS = $(am__append_1) -release `cat libtool-soversion` \
 	@SHARED_LDFLAGS@ $(am__empty)
--- readline/readline/config.h.in	2020-09-13 02:33:41.000000000 +0000
+++ readline/readline/config.h.in	2021-01-21 21:52:55.270000000 +0000
@@ -281,7 +281,7 @@
 
 #undef HAVE_BSD_SIGNALS
 
-#undef HAVE_POSIX_SIGNALS
+#define HAVE_POSIX_SIGNALS
 
 #undef HAVE_USG_SIGHOLD
 
--- ./libiberty/strsignal.c.orig	2021-01-21 20:42:22.879695310 +0000
+++ ./libiberty/strsignal.c	2021-01-21 20:45:50.325507224 +0000
@@ -549,7 +549,7 @@
 */
 
 #ifndef HAVE_PSIGNAL
-
+/*
 void
 psignal (int signo, char *message)
 {
@@ -566,7 +566,7 @@
       fprintf (stderr, "%s: %s\n", message, sys_siglist[signo]);
     }
 }
-
+*/
 #endif	/* ! HAVE_PSIGNAL */
 
 
--- gnulib/import/getcwd.c
+++ gnulib/import/getcwd.c
@@ -473,6 +473,8 @@ __getcwd (char *buf, size_t size)
   return NULL;
 }
 
+/*
 #ifdef weak_alias
 weak_alias (__getcwd, getcwd)
 #endif
+*/
\ No newline at end of file
--- gnulib/import/getrandom.c
+++ gnulib/import/getrandom.c
@@ -18,7 +18,7 @@
 /* Written by Paul Eggert.  */
 
 #include <config.h>
-
+#include <stddef.h>
 #include <sys/random.h>
 
 #include <errno.h>
--- sim/common/Make-common.in
+++ sim/common/Make-common.in
@@ -283,9 +283,9 @@ gentmap: Makefile $(srccom)/gentmap.c $(srccom)/nltvals.def
 targ-vals.h targ-map.c: stamp-tvals
 stamp-tvals: gentmap
 	rm -f tmp-tvals.h tmp-tmap.c
-	./gentmap -h >tmp-tvals.h
+	node ./gentmap -h >tmp-tvals.h
 	$(SHELL) $(srcroot)/move-if-change tmp-tvals.h targ-vals.h
-	./gentmap -c >tmp-tmap.c
+	node ./gentmap -c >tmp-tmap.c
 	$(SHELL) $(srcroot)/move-if-change tmp-tmap.c targ-map.c
 	touch stamp-tvals
 
